goal_x: 6
goal_y: 4
